### DISCUSSION TITLE
Missing Credentials Fields?

### DISCUSSION THREAD

**Discussion by vip3rousmango** [requarks/wiki#5706] [Help / Questions] [answered]
Hello!

I recently had to recover an expired domain and in doing so I believe the Let's Encrypt SSL isn't working.
In trying to log in the credentials field for the local user type is missing, so I'm not sure how to login as my admin account.

Is there something I can do to log back in as my admin account to reset the Let's Encrypt? Is this something I can run from a terminal for wiki.js to reset to allow that to re-appear (1GB droplet on DigitalOcean)?

![missing-credentials-fields](https://user-images.githubusercontent.com/514211/193609328-505ecbd5-2bd3-4d82-84c1-37012722d07d.png)

**Comment by NGPixel** [comment, marked as answer]
Add `?all=1` to the URL. You can't see it because you've set it to not show by default.

**Reply by vip3rousmango** [reply]
Thank you @NGPixel !! I'm back in. I appreciate the support. 
Loving Wiki,js as my go-to wiki platform.
